templates/Pages/cfs_isc23.php

```diff
@@ -0,0 +1,102 @@
+    <p>
+        Following the success of the Top500 in collecting and analyzing historical trends in supercomputer technology and evolution, the IO500 was created in 2017, published its first list at SC17, and has grown continually since then. The need for such an initiative has long been known within High-Performance Computing; however, defining appropriate benchmarks has long been challenging. Despite this challenge, the community, after long and spirited discussion, finally reached consensus on a suite of benchmarks and a metric for resolving the scores into a single ranking.
+    </p>
+
+    <p>
+        The multi-fold goals of the benchmark suite are as follows:
+    </p>
+
+    <ol>
+        <li>Maximizing simplicity in running the benchmark suite</li>
+        <li>Encouraging optimization and documentation of tuning parameters for performance</li>
+        <li>Allowing submitters to highlight their "hero run" performance numbers</li>
+        <li>Forcing submitters to simultaneously report performance for challenging IO patterns.</li>
+    </ol>
+
+    <p>
+        Specifically, the benchmark suite includes a hero-run of both IOR and mdtest configured however possible to maximize performance and establish an upper-bound for performance. It also includes an IOR and mdtest run with highly prescribed parameters in an attempt to determine a lower performance bound. Finally, it includes a namespace search as this has been determined to be a highly sought-after feature in HPC storage systems that has historically not been well-measured. Submitters are encouraged to share their tuning insights for publication.
+    </p>
+
+    <p>
+        The goals of the community are also multi-fold:
+    </p>
+
+    <ol>
+        <li>Gather historical data for the sake of analysis and to aid predictions of storage futures</li>
+        <li>Collect tuning information to share valuable performance optimizations across the community</li>
+        <li>Encourage vendors and designers to optimize for workloads beyond "hero runs"</li>
+        <li>Establish bounded expectations for users, procurers, and administrators</li>
+    </ol>
+
+    <h3>10 Client Node I/O Challenge</h3>
+
+    <p>
+        The 10 Client Node Challenge is conducted using the regular IO500 benchmark, however, with the rule that exactly 10 client nodes must be used to run the benchmark. You may use any shared storage with any number of servers. We will announce the results in the Production and Research lists as well as in separate derived lists.
+    </p>
+
+    <h3>Birds-of-a-Feather</h3>
+
+    <p>
+        Once again, we encourage you to
+        <?php echo $this->Html->link('submit',
+            [ 'controller' => 'pages', 'action' => 'display', 'submission' ],
+            [ 'class' => 'link' ]);
+         ?>
+        to join our community, and to attend the  
+        <?php echo $this->Html->link('ISC23 BoF',
+            [ 'controller' => 'pages', 'action' => 'display', 'bof-sc22' ],
```

Comment:
should be the ISC23 bof page

Reply:
@gflofst we do not have that page yet. I'll copy from last year, but let me know what changes need to be made.

Reply:
I'm getting a server error at the moment for io500.org. Any ideas?

Reply:
@gflofst I just cleared the cache, it looks ok. Please let me know if you still run into issues.

Reply:
@jeanbez website is back up. I'll get a placeholder page up for ISC23 and then we can get it live and linked in.